### PR TITLE
WIKI-1317 : upgrade smartgwt to 6.0

### PR DESCRIPTION
**Pull request by mrhannachi**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
     <com.ibm.icu.version>56.1</com.ibm.icu.version>
     <com.lowagie.itext.version>2.1.7</com.lowagie.itext.version>
     <com.mattbertolini.version>2.0.0</com.mattbertolini.version>
-    <com.smartgwt.version>4.0</com.smartgwt.version>
+    <com.smartgwt.version>6.0</com.smartgwt.version>
```
Comment (mgreau) [inline pom.xml]: As explained in [DEP-153](https://jira.exoplatform.org/browse/DEP-153) , the Maven coordinates have changed.
Please update this dependency with 
```
<groupId>com.isomorphic.smartgwt.lgpl</groupId>
<artifactId>smartgwt-lgpl</artifactId>
<version>6.0-p20170514</version>
```

So do not forget to:

1. rename the Maven property to `com.isomorphic.smartgwt.lgpl.version`
2. update the coordinates with the new version / groupId / artifactId
3. re-order both maven property and maven dependency declaration to keep the alphabetical order

thanks

**Review by mgreau — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 2]
```diff
@@ -205,9 +205,9 @@
         <version>${com.mattbertolini.version}</version>
       </dependency>
       <dependency>
-        <groupId>com.smartgwt</groupId>
-        <artifactId>smartgwt</artifactId>
-        <version>${com.smartgwt.version}</version>
+        <groupId>com.isomorphic.smartgwt.lgpl</groupId>
```
Comment (mgreau) [inline pom.xml]: Please re-order the maven dependency declaration to keep the alphabetical order by groupId

File: pom.xml [hunk 3]
```diff
@@ -57,7 +57,7 @@
     <com.ibm.icu.version>56.1</com.ibm.icu.version>
     <com.lowagie.itext.version>2.1.7</com.lowagie.itext.version>
     <com.mattbertolini.version>2.0.0</com.mattbertolini.version>
-    <com.smartgwt.version>4.0</com.smartgwt.version>
+    <com.isomorphic.smartgwt.lgpl.version>6.0-p20170514</com.isomorphic.smartgwt.lgpl.version>
```
Comment (mgreau) [inline pom.xml]: Please re-order the maven property  to keep the alphabetical order

**Review by mgreau — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (mrhannachi) [reply pom.xml]: done

File: pom.xml (hunk 2 above)
Comment (mrhannachi) [reply pom.xml]: done

**Review by mgreau — COMMENTED** [review]
Thanks, please squash the commits so that I can merge this PR.

**Comment by mrhannachi** [comment]
@mgreau commits squashed

**Review by mgreau — APPROVED** [review]

**Pull request merged**